multisig/simulations/sepolia/vip-025/index.ts

```diff
@@ -0,0 +1,112 @@
+  let vFRAXContract: Contract;
+  let vsFRAXContract: Contract;
+  let comptroller: Contract;
+  let rewardDistributor: Contract;
+  let xvs: Contract;
+
+  before(async () => {
+    resilientOracle = await ethers.getContractAt(RESILIENT_ORACLE_ABI, sepolia.RESILIENT_ORACLE);
+    poolRegistry = await ethers.getContractAt(POOL_REGISTRY_ABI, sepolia.POOL_REGISTRY);
+    vFRAXContract = await ethers.getContractAt(VTOKEN_ABI, vFRAX);
+    vsFRAXContract = await ethers.getContractAt(VTOKEN_ABI, vsFRAX);
+    comptroller = await ethers.getContractAt(COMPTROLLER_ABI, COMPTROLLER);
+    rewardDistributor = await ethers.getContractAt(REWARD_DISTRIBUTOR_ABI, REWARDS_DISTRIBUTOR_XVS);
+    xvs = await ethers.getContractAt(ERC20_ABI, XVS);
+  });
+
+  describe("Pre-VIP behavior", () => {
+    it("check FRAX price", async () => {
+      await expect(resilientOracle.getPrice(FRAX)).to.be.reverted;
+    });
+
+    it("check sFRAX price", async () => {
+      await expect(resilientOracle.getPrice(sFRAX)).to.be.reverted;
+    });
+  });
+
+  describe("Post-VIP behavior", async () => {
+    before(async () => {
+      await pretendExecutingVip(vip025());
+    });
+
+    it("check FRAX price", async () => {
+      expect(await resilientOracle.getPrice(FRAX)).to.equal(parseUnits("1", 18));
+    });
+
+    it("check sFRAX price", async () => {
+      expect(await resilientOracle.getPrice(sFRAX)).to.equal(SFRAX_TO_FRAX_RATE);
+    });
+
+    it("should have 10 markets in core pool", async () => {
+      const poolVTokens = await comptroller.getAllMarkets();
+      expect(poolVTokens).to.have.lengthOf(10);
+    });
+
+    it("should add vFRAX to the pool", async () => {
+      const registeredVToken = await poolRegistry.getVTokenForAsset(comptroller.address, FRAX);
+      expect(registeredVToken).to.equal(vFRAX);
+    });
+
+    it("should add vsFRAX to the pool", async () => {
+      const registeredVToken = await poolRegistry.getVTokenForAsset(comptroller.address, sFRAX);
+      expect(registeredVToken).to.equal(vsFRAX);
+    });
+
+    it("check vFRAX ownership", async () => {
+      expect(await vFRAXContract.owner()).to.equal(sepolia.GUARDIAN);
+    });
+
+    it("check vsFRAX ownership", async () => {
+      expect(await vsFRAXContract.owner()).to.equal(sepolia.GUARDIAN);
+    });
+
+    it("check vFRAX supply", async () => {
+      const expectedSupply = parseUnits("5", 11);
+      expect(await vFRAXContract.balanceOf(sepolia.VTREASURY)).to.equal(expectedSupply);
+    });
+
+    it("check vsFRAX supply", async () => {
+      const expectedSupply = parseUnits("48", 10);
+      expect(await vsFRAXContract.totalSupply()).to.equal(expectedSupply);
+    });
+
+    it("check reward token", async () => {
+      expect(await rewardDistributor.rewardToken()).to.equal(XVS);
+    });
+
+    it(`rewards distributor should have balance`, async () => {
+      expect(await xvs.balanceOf(rewardDistributor.address)).to.gte(XVS_REWARD_TRANSFER);
+    });
+  });
```

Comment:
We could add some of the shared checks we have (IR, RewardsDistributor, VToken, Isolated pool)